pkg/types/types.go

```diff
@@ -50,7 +51,10 @@ type VM struct {
 	// Tags to be applied to the VM
 	Tags     map[string]string `yaml:"tags,omitempty"`
 	Commands []string          `yaml:"commands,omitempty"`
-	IP       string            `yaml:"-"`
+	// KonfigadmFile allows for the specification of a konfigadm
+	// spec used as basis for configuring the VM on creation.
```

Comment:
```suggestion
	// A path to a konfigadm specification used for configuring the VM on creation.
```

`{Fieldname} comment` is go convention to make godocs readable at a summary level, but the docs we create from these comments are in a table with the Fieldname as the first column so adding it is suplerfous